guides/common/modules/ref_ipxe-bootloaders.adoc

```diff
@@ -0,0 +1,125 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="ref_ipxe-bootloaders_{context}"]
+= iPXE / GRUB2 UEFI HTTP(S) boot loaders
+
+This section describes HTTP-based boot methods such as *iPXE*, *iPXE chainloading* and *GRUB2 UEFI HTTP(S) Boot*, which use the {SmartProxy}’s HTTPBoot module to deliver bootloaders and installation files.
+
+== Overview
+
+{Project} also supports boot methods that use *HTTP* instead of the traditional *TFTP-based PXE* mechanism.
+These include *iPXE*, *iPXE chainloading* and *UEFI HTTP Boot*, both of which leverage the {SmartProxy}’s HTTPBoot module to deliver bootloaders and installation files.
```

Comment:
```suggestion
This section describes HTTP-based boot methods such as *iPXE*, *iPXE chainloading* and *GRUB2 UEFI HTTP(S) Boot*, which use the {SmartProxy}’s HTTPBoot module to deliver bootloaders and installation files.

In addition to the *TFTP-based PXE* mechanism, {Project} supports boot methods that use *HTTP*.
Boot methods that use HTTP  include *iPXE*, *iPXE chainloading*, and *UEFI HTTP Boot*, both of which leverage the {SmartProxy}’s HTTPBoot module to deliver bootloaders and installation files.
```

This needs a bit more clarification, not sure how to align three items being referred to as "both". Will investigate more.